uber/templates/registration/register.html

```diff
@@ -1,131 +1,64 @@
-<!DOCTYPE HTML>
```

Comment:
> In the future, we may want to take this out of the main codebase and have it as a standard part of the plugins along with showing/hiding things on the pre-reg page. I don't know if there's differences between MAG events, but e.g. AnthroCon sells donation levels ATD.

Oh, interesting.  In that case, I'll make one last change before anyone merges this to make it so that the "hide donation levels" can be overridden in the ``regextras.html`` override file.